scanner/indexer/service.go

```diff
@@ -37,6 +38,12 @@ func (s *indexerService) RegisterServiceServer(grpcServer *grpc.Server) {
 	v4.RegisterIndexerServer(grpcServer, s)
 }
 
+// AuthFuncOverride specifies the auth criteria for this API.
+func (s *indexerService) AuthFuncOverride(ctx context.Context, fullMethodName string) (context.Context, error) {
+	// TODO: Setup permissions for indexer.
+	return ctx, allow.Anonymous().Authorized(ctx, fullMethodName)
```

Comment:
Recommend adding a failing test for `anonymous` access just in case this `TODO` is missed, that way an unauthenticated API is less likely to be released.

Reply:
I appreciate your suggestion, but a failing test may disrupt development and encourage allowing non-green test statuses elsewhere. I created a tracking ticket to ensure we don't overlook adding auth later: https://issues.redhat.com/browse/ROX-18526